src/Process/FeatureProcessing.js

```diff
@@ -7,29 +7,13 @@ import Extent from 'Core/Geographic/Extent';
 
 const _extent = new Extent('EPSG:4326', 0, 0, 0, 0);
 const coord = new Coordinates('EPSG:4326', 0, 0, 0);
-const vector = new THREE.Vector3();
-const tmp = new THREE.Vector3();
+const mat4 = new THREE.Matrix4();
 
-function applyOffset(obj, offset, quaternion, offsetAltitude = 0) {
+function applyMatrix4(obj, mat4) {
```

Comment:
Good to have this, however you forgot to adapt the `source_stream_wfs_3d` example with the altitude offset.